wsd/DocumentBroker.hpp

```diff
@@ -497,8 +497,11 @@ class DocumentBroker : public std::enable_shared_from_this<DocumentBroker>
     void removeEmbeddedMedia(const std::string& json);
 
 private:
-    /// get the session id of a session that can write the document for save / locking.
+    /// Get the session that can write the document for save / locking / uploading.
     /// Note that if there is no loaded and writable session, the first will be returned.
+    std::shared_ptr<ClientSession> getWriteableSession() const;
```

Comment:
This is to avoid doing `_sessions.find(sessionId)` use the ClientSession instance direactly.